dbt/include/snowflake/macros/relations/table/create.sql

```diff
@@ -26,7 +20,16 @@
 
     {{ sql_header if sql_header is not none }}
 
-        create or replace {{ table_type }} table {{ relation }}
+        create or replace {{ materialization_prefix }} table {{ relation }}
+        {%- if relation.is_iceberg_format %}
+          {#
+            Valid DDL in CTAS statements. Plain create statements have a different order.
```

Comment:
The order of the options matters? That's wild. Also, I think you can drop the `{% else %}` below since there's nothing in the block.

Reply:
yeah there's an alternate CREATE TABLE where the params are in a different place entirely.

And yep that's a leftover artifact. Removed!